fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

```diff
@@ -760,6 +769,19 @@ public void gsonPreProcess() throws IOException {
         }
     }
 
+    public String getConvertedColumnName() {
```

Comment:
```suggestion
    public String getPhysicalName() {
```
I think use getPhysicalName is more clear, because when physicalName is null, name is also physicalName.